### ISSUE TITLE
Inconsistent URLs

### ISSUE THREAD

**Issue by jimafisk** [CityOfBoston/boston.gov-d8#635]
D8: http://bostond8dev.prod.acquia-sites.com/latest-city-boston-news
D7: https://www.boston.gov/news

Potentially related to pathauto: https://github.com/CityOfBoston/boston.gov-d8/issues/431

**Comment by jimafisk** [comment]
More examples...

D8: http://bostond8dev.prod.acquia-sites.com/elections/how-add-your-name-annual-census
D7: https://www.boston.gov/departments/elections/how-add-your-name-annual-census

D8: http://bostond8dev.prod.acquia-sites.com/elections/how-register-vote
D7: https://www.boston.gov/departments/elections/how-register-vote

D8: http://bostond8dev.prod.acquia-sites.com/how-vote-absentee-ballot
D7: https://www.boston.gov/departments/elections/how-vote-absentee-ballot

**Comment by jimafisk** [comment]
D8: http://bostond8dev.prod.acquia-sites.com/health-and-human-services/bcyf-city-hall-childcare
D7: https://www.boston.gov/departments/boston-centers-youth-families/bcyf-city-hall-childcare

D8: http://bostond8dev.prod.acquia-sites.com/health-and-human-services/bcyf-family-gym
D7: https://www.boston.gov/departments/boston-centers-youth-families/bcyf-family-gym

D8: http://bostond8dev.prod.acquia-sites.com/health-and-human-services/bcyf-girls
D7: https://www.boston.gov/departments/boston-centers-youth-families/bcyf-girls

Basically any subpage off a department page is missing the `/departments` section in the path.

**Comment by davidrkupton** [comment]
Also:
D8: http://bostond8dev.prod.acquia-sites.com/downtown-parking-freeze
D7: https://www.boston.gov/departments/environment/air-pollution-control-commission/downtown-parking-freeze

**Comment by davidrkupton** [comment]
Duplicate of #519.  
#519 closed.

**Comment by reillyzlab** [comment]
Drupal 8: bostond8dev.prod.acquia-sites.com/trash-and-recycling-0
Drupal 7: boston.gov/trash-and-recycling-guide

**Comment by andrebtlima** [comment]
Another example:
http://bostond8.prod.acquia-sites.com/contact-1/test-page

and on D7
https://www.boston.gov/departments/digital-team/test-page#youtube-video

**Comment by reillyzlab** [comment]
Test if this is working for:

- [ ] Pre-existing URLs/pages (do they match what exists?)
- [ ] New URLs/pages (are they accurately getting created?)

**Comment by andrebtlima** [comment]
Resolved so closing.
